src/Conversations/Conversation.php

```diff
@@ -6,14 +6,16 @@
 use Psr\SimpleCache\InvalidArgumentException;
 use RuntimeException;
 use SergiX44\Nutgram\Nutgram;
+use SergiX44\Nutgram\Telegram\Properties\UpdateType;
 
 /**
  * Class Conversation
  * @package SergiX44\Nutgram\Conversation
  */
 abstract class Conversation
 {
-    protected bool $skipHandlers = false;
+    /** @var bool|UpdateType[] */
+    protected mixed $skipHandlers = false;
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏instead of mixed I think should be array|bool

Reply:
This is a BC for those who have overridden this property.